### ISSUE TITLE
kgriffs

### ISSUE THREAD

**Issue by dmproia** [falconry/falcon#1725]
Should I reveal the original source code with the original author and his license or do you want to pay some respect? This is ludicrous as you stole my code and then kept blocking me out of my server. You know what you did and the hacks to try and keep a secret as I’m not going away. You never had permission to use my code and it was licensed and registered to me. I made this fucking product not you. You didn’t even have the decency to change the name of the project. You are a toolbox.

**Comment by vytas7** [comment]
Hello @dmproia ,
AFAIC Falcon development has been carried out fully in the open since 2013, so the alleged code appropriation must have happened prior to that date. I seriously doubt @kgriffs had superpowers to keep you blocking for 7 years, unless you were imprisoned in some basement.

TL;DR [Who you jiving with that Cosmik Debris? Don't you waste your time on us.](https://www.youtube.com/watch?v=Dp6LT2MdaPI&feature=youtu.be&t=40)

**Comment by jmvrbanac** [comment]
Falcon has a well known open-source track record since 2013. Back then it was built and primarily used by Rackspace for internal projects and was licensed as Apache v2 for open-source development. Outside of this repository, you can also find discussion back from 2013, on the Openstack mailing lists regarding the potential usage of Falcon in Openstack projects as well.

If you have specific concerns, please express them in a manner in accordance to our [code of conduct](https://github.com/falconry/falcon/blob/master/CODEOFCONDUCT.md).
